common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -104,6 +104,8 @@ public enum Feature {
         MULTI_SITE("Multi-site support", Type.PREVIEW),
 
         OFFLINE_SESSION_PRELOADING("Offline session preloading", Type.DEPRECATED),
+
+        UI_CUSTOMIZATION("UI customization spi", Type.EXPERIMENTAL),
```

Comment:
Didn't we have another name for this?  Maybe it was something like, "Declarative UI?"

UI_CUSTOMIZATION is too broad of a term to describe what this does as there are other ways to customize the UI.